Automattic/wp-calypso
# WooCommerce: Add Product Categories

Update: This PR has 4 others rolled up into it.

To Test:
1. run `make test` and verify all tests pass.
2. `make run`
3. Visit "http://calypso.localhost:3000/store/products/<site url>/add"
4. Verify you can edit all fields and set values on them.

Original base PR:

This adds the base reducer for WooCommerce API data and sorts the data by siteId.
It also adds code to test via the error actions/reducer.

To Test:
1. run `make test` and verify all tests pass.
